### PR TITLE
Adding allHorizontal/anyHorizontal

### PR DESCRIPTION
**Pull request by Icequeen3333**
Fixed the problem in #95, where anyHorizontal/allHorizontal did not exist. Since sumHorizontal, minHorizonal/maxHorizontal also do not exist and are in the same main commit, I will add them when I figure out how to fix it.

### CODE REVIEW

File: polars/lazy/functions.ts [hunk 1]
```diff
@@ -630,9 +630,90 @@ export function struct(
...
+    Examples
+    --------
+    >>> const df = pl.DataFrame(
+    ...     {
+    ...         "a": [False, False, True, True],
+    ...         "b": [False, True, None, True],
+    ...         "c": ["w", "x", "y", "z"],
+    ...     }
+    ... )
+    >>> df.with_columns(pl.all_horizontal("a", "b"))
```
Comment (universalmind303) [inline polars/lazy/functions.ts:655]: looks like you missed updating some variables to JS syntax

File: polars/lazy/functions.ts [hunk 2]
```diff
@@ -630,9 +630,90 @@ export function struct(
...
+    Examples
+    --------
+    >>> df = pl.DataFrame(
+    ...     {
+    ...         "a": [False, False, True, None],
+    ...         "b": [False, True, None, None],
+    ...         "c": ["w", "x", "y", "z"],
+    ...     }
+    ... )
+    >>> df.with_columns(pl.any_horizontal("a", "b"))
```
Comment (universalmind303) [inline polars/lazy/functions.ts:695]: update variables & functions to JS

File: src/lazy/dsl.rs [hunk 3]
```diff
@@ -1662,3 +1662,17 @@ pub fn as_struct(exprs: Vec<&JsExpr>) -> JsExpr {
     let exprs = exprs.to_exprs();
     polars::lazy::dsl::as_struct(&exprs).into()
 }
+
+//
```
Comment (universalmind303) [inline src/lazy/dsl.rs:1666]: remove

**Review by universalmind303 — CHANGES_REQUESTED** [review]

File: polars/lazy/functions.ts (hunk 1 above)
Comment (Icequeen3333) [reply polars/lazy/functions.ts]: Yep, used them for examples and then forgot that JS and python are in fact two different languages when it comes to syntax on true/false/null, fixed now

**Pull request merged**